v2/apierror/api_error.go

```diff
@@ -77,7 +81,7 @@ func (a *APIError) Error() string {
 	d.WriteString(a.err.Error() + "\n")
 
 	if a.details.ErrorInfo != nil {
-		d.WriteString(fmt.Sprintf("error details: name = ErrorInfo reason = %s domain = %s metadata = %s",
+		d.WriteString(fmt.Sprintf("error details: name = ErrorInfo reason = %s domain = %s metadata = %s \n",
```

Comment:
Nit: please remove the space between `%s \n`. Applies to all such format strings